packages/core/addon/components/visualization-config/wrapper.js

```diff
@@ -22,11 +22,19 @@ export default Component.extend({
 
   classNames: ['visualization-config'],
 
+  /**
+   * @property {String} seriesPath
+   */
+  seriesPath: computed('visualization.type', function() {
+    return get(this, 'visualization.type') === 'pie-chart' ? 'series' : 'axis.y.series';
```

Comment:
Is there a way we can make the wrapper unaware of concrete visualization types?

Reply:
So all of these props and methods are here because moving the metric selector above other line/bar options mixed up things a little.
Inside the wrapper we're now rendering:
- metric selector **if series and of type dimension**
- specific chart config by type
- series add/remove **if series and of type dimension**

We _could_ delegate all of the metric selector logic from the wrapper back to `chart-type/dimension` (as it was before) and in line/bar/pie chart do something like 
```
{{#component
  (concat typePrefix seriesType) <-- e.g. visualization-config/chart-type/dimension
  ...
}}
   {{!-- specific line/bar chart options --}}
{{/component}}
```

while `visualization-config/chart-type/dimension` would be something like
```
{{#if showMetricSelect}}
  <div class="metric-select">...</div>
{{/if}}
{{yield}}
{{chart-series-collection   <-- add/remove series
    ...
}}
```

The other option which feels hacky to me would be to keep it in the wrapper and just check for `visualization.metadata.series` or `visualization.metadata.axis.y.series` to determine whether it's a series without relying on chart type.

I'm not sure about either, would like to hear your suggestion and I'll see if I can come up with something more generic.

Also keep in mind that currently `visualization-config/chart-type/date-time` and `visualization-config/chart-type/metric` don't render anything at all.